packages/xchain-ethereum/CHANGELOG.md

```diff
@@ -1,3 +1,13 @@
+# v.x.x.x
+
+### Update
+
+- Add `estimateCall` to estimate gaslimit for `call` function.
+
+### Fix
+
+- Fix gas limit estimation logic.
```

Comment:
Breaking change (types of `estimateGasLimit()` + `FeesWithGasPricesAndLimits` have been changed)